test/acquisition/test_analytic.py

```diff
@@ -291,6 +292,47 @@ def test_posterior_mean_batch(self):
                 PosteriorMean(model=mm2)
 
 
+class TestPosteriorStandardDeviation(BotorchTestCase):
+    def test_posterior_stddev(self):
+        for dtype in (torch.float, torch.double):
+            mean = torch.rand(3, 1, device=self.device, dtype=dtype)
+            std = torch.rand_like(mean)
+            mm = MockModel(MockPosterior(mean=mean, variance=std.square()))
+
+            module = PosteriorStandardDeviation(model=mm)
```

Comment:
Nit: It would be more conventional and clearer to rename `module` to `acqf`, since `module` is very generic